src/core/pipe/pipe.column.js

```diff
@@ -119,6 +118,11 @@ function addPivotColumns(columns, heads) {
 	return rows;
 }
 
+function isShowCheckboxes(model){
```

Comment:
rename it to canAddSelectColumn or something like that, in reality column doesn't know anything about checkboxes anything